### ISSUE TITLE
Can I use seurat to analyze the data after MNNs batch normalized?

### ISSUE THREAD

**Issue by hua1991** [satijalab/seurat#3114]
Dear all,
I want to re-analyze some data with Seurat. The data compromise several subjects, it was combined and batch normalized using MNNs (mnnpy). I'm not sure whether this kind of data after batch normalized can be used for Seurat analysis? If yes, which Seurat tutorial should I follow? Thanks.

**Comment by satijalab** [comment]
Yes - you will need to convert the python output (I assume this is in scanpy) to a Seurat object, you can use the converters here:
https://github.com/mojaveazure/seurat-disk

**Comment by hua1991** [comment]
Sorry to open this question again, but I still can't figure it out. The data I can only have is a .txt file of batch normalized data, without the raw count data. I tried to setup seurat object with this txt file, but the tsne plot had an obvious batch effect. I also tried to extract each subject from the txt file, then integrated them to follow the "PBMC control vs stimulated" tutorial, this time batch effect was gone, but the "IntegrateData" step also removed the batch effect by CAA method, the data already batch effect removed, why does it still need to remove the batch effect. And I'm still not sure whether it is plausible to setup seurat object using batch normalized data. Therefore, I'm confused about whether I can use this data (without raw count data) to compare it with my own data.
